src/Compilers/CSharp/Test/Emit3/RefStructInterfacesTests.cs

```diff
@@ -5853,9 +5853,9 @@ void verify(ModuleSymbol m)
                 Assert.Equal("I1", s1.InterfacesNoUseSiteDiagnostics().Single().ToTestDisplayString());
             }
 
-            CreateCompilation(src, targetFramework: TargetFramework.Net80, parseOptions: TestOptions.RegularNext).VerifyDiagnostics();
+            CreateCompilation(src, targetFramework: TargetFramework.Net70, parseOptions: TestOptions.RegularNext).VerifyDiagnostics();
```

Comment:
>TargetFramework.Net70 [](http://example.com/codeflow?start=52&length=21)

Was there a reason to change this test to .NET7?

Reply:
> Was there a reason to change this test to .NET7?

Just to make the scenario "stronger"